<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="PoliqarpWidget" name="poliqarpWidget" native="true"/>
      <widget class="QDjVuWidget" name="djvuWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionFileOpen"/>
    <addaction name="actionFileOpenLocation"/>
    <addaction name="actionFileRecent"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelpAbout"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>&amp;Zoom</string>
     </property>
     <addaction name="actionZoomIn"/>
     <addaction name="actionZoomOut"/>
     <addaction name="separator"/>
     <addaction name="actionZoomOneToOne"/>
     <addaction name="actionZoomFitWidth"/>
     <addaction name="actionZoomFitPage"/>
     <addaction name="separator"/>
     <addaction name="actionZoom300"/>
     <addaction name="actionZoom200"/>
     <addaction name="actionZoom150"/>
     <addaction name="actionZoom100"/>
     <addaction name="actionZoom75"/>
     <addaction name="actionZoom50"/>
    </widget>
    <widget class="QMenu" name="menuRotate">
     <property name="title">
      <string>&amp;Rotate</string>
     </property>
     <addaction name="actionRotateLeft"/>
     <addaction name="actionRotateRight"/>
     <addaction name="separator"/>
     <addaction name="actionRotate0"/>
     <addaction name="actionRotate90"/>
     <addaction name="actionRotate180"/>
     <addaction name="actionRotate270"/>
    </widget>
    <addaction name="menuZoom"/>
    <addaction name="menuRotate"/>
    <addaction name="actionViewContinuous"/>
    <addaction name="actionViewSideBySide"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuGo">
    <property name="title">
     <string>&amp;Go</string>
    </property>
    <addaction name="actionGoToFirstPage"/>
    <addaction name="actionGoToLastPage"/>
    <addaction name="actionGoToNextPage"/>
    <addaction name="actionGoToPreviousPage"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionViewSidebar"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_View"/>
   <addaction name="menuGo"/>
   <addaction name="menu_Settings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFileOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFileRecent">
   <property name="text">
    <string>Open recent</string>
   </property>
   <property name="toolTip">
    <string>Open recent</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>Zoom &amp;in</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>Zoom &amp;out</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionZoomOneToOne">
   <property name="text">
    <string>One &amp;to one</string>
   </property>
  </action>
  <action name="actionZoomFitWidth">
   <property name="text">
    <string>Fit &amp;Width</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
  </action>
  <action name="actionZoomFitPage">
   <property name="text">
    <string>Fit &amp;Page</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionZoom300">
   <property name="text">
    <string>&amp;300%</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionZoom200">
   <property name="text">
    <string>&amp;200%</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionZoom150">
   <property name="text">
    <string>150%</string>
   </property>
  </action>
  <action name="actionZoom100">
   <property name="text">
    <string>&amp;100%</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionZoom75">
   <property name="text">
    <string>&amp;75%</string>
   </property>
  </action>
  <action name="actionZoom50">
   <property name="text">
    <string>&amp;50%</string>
   </property>
  </action>
  <action name="actionRotateLeft">
   <property name="text">
    <string>Rotate &amp;left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="actionRotateRight">
   <property name="text">
    <string>Rotate &amp;right</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionRotate90">
   <property name="text">
    <string>Rotate &amp;90째</string>
   </property>
  </action>
  <action name="actionRotate180">
   <property name="text">
    <string>Rotate &amp;180째</string>
   </property>
  </action>
  <action name="actionRotate270">
   <property name="text">
    <string>Rotate &amp;270째</string>
   </property>
  </action>
  <action name="actionRotate0">
   <property name="text">
    <string>Rotate &amp;0째</string>
   </property>
  </action>
  <action name="actionViewContinuous">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Continuous</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionViewSideBySide">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Side by side</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionGoToFirstPage">
   <property name="text">
    <string>&amp;First page</string>
   </property>
  </action>
  <action name="actionGoToLastPage">
   <property name="text">
    <string>&amp;Last page</string>
   </property>
  </action>
  <action name="actionGoToNextPage">
   <property name="text">
    <string>&amp;Next page</string>
   </property>
  </action>
  <action name="actionGoToPreviousPage">
   <property name="text">
    <string>Previous page</string>
   </property>
  </action>
  <action name="actionGoBackward">
   <property name="text">
    <string>&amp;Backward</string>
   </property>
  </action>
  <action name="actionGoForward">
   <property name="text">
    <string>&amp;Forward</string>
   </property>
  </action>
  <action name="actionFileOpenLocation">
   <property name="text">
    <string>Open &amp;location</string>
   </property>
   <property name="toolTip">
    <string>Open location</string>
   </property>
  </action>
  <action name="actionViewSidebar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View &amp;sidebar</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QDjVuWidget</class>
   <extends>QWidget</extends>
   <header>qdjvuwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PoliqarpWidget</class>
   <extends>QWidget</extends>
   <header>poliqarpwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionViewSidebar</sender>
   <signal>toggled(bool)</signal>
   <receiver>poliqarpWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
